server/src/FeatureFinder.ts

```diff
@@ -83,53 +101,62 @@ export class FeatureFinder {
   }
 
   private async installAndFindCommandForV1(): Promise<Command | undefined> {
-    if (this.python === 'python') {
-      this.python = 'python3';
-    }
-
     try {
       await this.installLatestDbtRpc();
     } catch (e) {
       console.log('Error while installing dbt-rpc');
       return undefined;
     }
-
     return new DbtRpcCommand(FeatureFinder.DBT_RPC_PARAMS, this.python);
   }
 
-  private async findDbtRpcGlobalVersion(): Promise<DbtVersion | undefined> {
-    return this.findCommandVersion(new DbtRpcCommand([FeatureFinder.VERSION_PARAM]));
+  private async findDbtRpcGlobalVersion(): Promise<DbtVersionInfo | undefined> {
+    return this.dbtProfileType ? this.findCommandVersion(new DbtRpcCommand([FeatureFinder.VERSION_PARAM]), this.dbtProfileType) : undefined;
```

Comment:
I don't understand this logic. Why if we don't have profile type specified we always have dbt global version `undefined`?